pubkey_encrypt.links.task.yml

```diff
@@ -0,0 +1,9 @@
+pubkey_encrypt.initialization_settings_tab:
+  route_name: pubkey_encrypt.initialization_settings
+  title: Initialize Module
```

Comment:
This is too generic of a title. What about PubKey Encrypt Initialization

Is this used by the batch API? Not sure why we have an initialization task defined here